doc/api/https.md

```diff
@@ -176,9 +176,14 @@ See [`server.maxHeadersCount`][] in the `node:http` module.
 
 <!-- YAML
 added: v14.11.0
+changes:
+  - version: REPLACEME
```

Comment:
I think you should replace this with the version #41263 was released into.

Reply:
Yup - https://github.com/nodejs/node/blob/main/doc/changelogs/CHANGELOG_V18.md#2022-04-19-version-1800-current-bethgriggs